volontulo/templates/common/badges.html

```diff
@@ -1,12 +1,11 @@
 {% if badges %}
-    <h2>Odznaki użytkownika</h2>
-    <ul class="badges">
+    <ul class="badges list-unstyled">
         {% for badge in badges %}
             <li>
-                <span class="name priority-{{ badge.badge__priority }}">{{ badge.badge__name }}</span> x <span class="badge">{{ badge.badges }}</span>
+                <b class="name priority-{{ badge.badge__priority }}">{{ badge.badge__name }}</b> <span class="badge">{{ badge.badges }}</span>
```

Comment:
My suggestion is to not using representative elements: b, i, u. It's legacy after firsts HTML stsandards. strong would be better for this.